./esbmc-wrapper.py -p ../sv-benchmarks/c/properties/unreach-call.prp -s kinduction --arch 64 ../sv-benchmarks/c/ldv-linux-3.14/linux-3.14_complex_emg_linux-kernel-locking-spinlock_fs-nfs-nfs_layout_nfsv41_files.cil.i


--------------------------------------------------------------------------------


Verifying with ESBMC
Command: ./esbmc --no-div-by-zero-check --force-malloc-success --state-hashing --no-align-check --k-step 2 --floatbv --unlimited-k-steps ../sv-benchmarks/c/ldv-linux-3.14/linux-3.14_complex_emg_linux-kernel-locking-spinlock_fs-nfs-nfs_layout_nfsv41_files.cil.i --64 --witness-output linux-3.14_complex_emg_linux-kernel-locking-spinlock_fs-nfs-nfs_layout_nfsv41_files.cil.i.graphml --no-pointer-check --no-bounds-check --interval-analysis --k-induction --max-inductive-step 3 

function call: argument "c:linux-3.14_complex_emg_linux-kernel-locking-spinlock_fs-nfs-nfs_layout_nfsv41_files.cil.i@465000@F@nfs41_setup_sequence@arg0" type mismatch: got code, expected pointer


Unknown
